test/fast/Unit tests/nvm_get_mirror

```diff
@@ -12,5 +12,17 @@ die () { echo "$@" ; exit 1; }
 ! nvm_get_mirror node foo || die 'unknown release type did not error'
 ! nvm_get_mirror iojs foo || die 'unknown release type did not error'
 
-[ "$(nvm_get_mirror node std)" = "${NVM_NODEJS_ORG_MIRROR}" ] || die 'node std did not yield NVM_NODEJS_ORG_MIRROR'
-[ "$(nvm_get_mirror iojs std)" = "${NVM_IOJS_ORG_MIRROR}" ] || die 'node std did not yield NVM_NODEJS_ORG_MIRROR'
+
+[ -z "$NVM_NODEJS_ORG_MIRROR" ] || die "MIRROR environment variables should no longer be exported"
+[ -z "$NVM_IOJS_ORG_MIRROR" ] || die "MIRROR environment variables should no longer be exported"
+
+[ "$(nvm_get_mirror node std)" = "https://nodejs.org/dist" ] || die "incorrect default node-std mirror"
+[ "$(nvm_get_mirror iojs std)" = "https://iojs.org/dist" ] || die "incorrect default iojs-std mirror"
+
+NVM_NODEJS_ORG_MIRROR="test://domain"
+[ "$(nvm_get_mirror node std)" = "test://domain" ] || die "node-std mirror should respect NVM_NODEJS_ORG_MIRROR"
+unset NVM_NODEJS_ORG_MIRROR
+
+NVM_IOJS_ORG_MIRROR="test://domain"
+[ "$(nvm_get_mirror iojs std)" = "test://domain" ] || die "iojs-std mirror should respect NVM_IOJS_ORG_MIRROR"
+unset NVM_IOJS_ORG_MIRROR
```

Comment:
it'd be great if this file had a trailing newline (i can add this myself tho, nbd)